plugins/woo-ai/src/product-description/product-description-button-container.tsx

```diff
@@ -67,7 +64,7 @@ export function WriteItForMeButtonContainer() {
 
 				setFetching( false );
 
-				if ( reason !== 'abort' ) {
+				if ( reason === 'finished' ) {
```

Comment:
Is reason being finished always a success? Should we check the content length or any other parameters to ensure that there was at least an expectation of a successful output before prompting for feedback.

Reply:
This is only fired after receiving a complete response with no errors, and I haven't seen an instance of this being fired in an instance that we wouldn't want to initiate feedback. Even if we received something odd, maybe we'd still want the negative feedback signal in that case?